main.go

```diff
@@ -44,6 +46,7 @@ var (
 	workers           = flag.Int("workers", 5, "Number of sync workers to run (default 5)")
 	eventsQPS         = flag.Float64("events-qps", 1./300., "Rate of events flowing per object (default - 1 event per 5 minutes)")
 	eventsBurst       = flag.Int("events-burst", 25, "Number of events allowed to send per object (default 25)")
+	pprofAddr         = flag.String("pprof-address", "0", "Enable pprof and bind to endpoint - /debug/pprof, set to 0 to disable pprof serviing")
```

Comment:
query: Does this need a doc update?

Reply:
@mjsmith1028 please update documentation as well, there is typo in `serviing` -> `serving`